databases/postgres-pgvector/manifests/01-basic-cluster/postgreSQL_cluster.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START gke_databases_postgresql_pgvector_01_postgresql_cluster]
```

Comment:
```suggestion
# [START gke_databases_postgres_pgvector_01_postgresql_cluster]
```

Reply:
Suggesting using the same prefix everywhere. I noticed 2 of the tags were "postgresql" instead of "postgres".